frontend/mr-admin-flate/src/pages/tiltaksgjennomforinger/TiltaksgjennomforingSkjemaPage.tsx

```diff
@@ -30,6 +31,7 @@ const TiltaksgjennomforingSkjemaPage = () => {
     TiltaksgjennomforingSchema,
     searchParams.get("utkastId") || undefined,
   );
+  const { data: ansatt, isLoading: isLoadingAnsatt } = useHentAnsatt();
```

Comment:
Hvis vi benytter `isPending` i stedet for `isLoading` så slipper vi "loading"-flicker, i tillegg så slipper vi nullability-sjekk på linje 63

Reply:
Litt mer info etter at vi gikk over til v5: https://tanstack.com/query/v5/docs/react/guides/migrating-to-v5#status-loading-has-been-changed-to-status-pending-and-isloading-has-been-changed-to-ispending-and-isinitialloading-has-now-been-renamed-to-isloading

Reply:
Ved å bruke `isPending` vil fortsatt typen til `data` være `NavAnsatt | undefined` så nullability-sjekken virker å måtte være der.